r/R/arrow-package.R

```diff
@@ -182,37 +182,50 @@ configure_tzdb <- function() {
 .onAttach <- function(libname, pkgname) {
   # Just to be extra safe, let's wrap this in a try();
   # we don't want a failed startup message to prevent the package from loading
-  try({
-    # On macOS only, Check if we are running in under emulation, and warn this will not work
-    if (on_rosetta()) {
-      packageStartupMessage(
-        paste(
-          "Warning:",
-          "  It appears that you are running R and Arrow in emulation (i.e. you're",
-          "  running an Intel version of R on a non-Intel mac). This configuration is",
-          "  not supported by arrow, you should install a native (arm64) build of R",
-          "  and use arrow with that. See https://cran.r-project.org/bin/macosx/",
-          "",
-          sep = "\n"
+  try(
+    {
```

Comment:
styler apparently demands this now?. I can back it out if we don't like it, but came from `make doc` so we'll be fighting it if we do.